### ISSUE TITLE
wavedrom doesn't generate

### ISSUE THREAD

**Issue by jnk0le** [jnk0le/XTightlyCoupledIO#2]

**Comment by jnk0le** [comment]
https://github.com/jnk0le/XTightlyCoupledIO/actions/runs/3609356314

ascidoctor-mathematical gem fails to install

**Comment by jnk0le** [comment]
```
  mathematical.c: At top level:
  cc1: note: unrecognized command-line option ‘-Wno-self-assign’ may have been
  intended to silence earlier diagnostics
  cc1: note: unrecognized command-line option ‘-Wno-parentheses-equality’ may have
  been intended to silence earlier diagnostics
  cc1: note: unrecognized command-line option ‘-Wno-constant-logical-operand’ may
  have been intended to silence earlier diagnostics
  linking shared-object mathematical/mathematical.so
  /usr/bin/ld: cannot find -lwebp: No such file or directory
  collect2: error: ld returned 1 exit status
  make: *** [Makefile:262: mathematical.so] Error 1
  
  make failed, exit code 2
  
  Gem files will remain installed in
  /home/runner/work/XTightlyCoupledIO/XTightlyCoupledIO/vendor/bundle/ruby/2.6.0/gems/mathematical-1.6.14
  for inspection.
  Results logged to
  /home/runner/work/XTightlyCoupledIO/XTightlyCoupledIO/vendor/bundle/ruby/2.6.0/extensions/x86_64-linux/2.6.0/mathematical-1.6.14/gem_make.out
  
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/2.6.0/rubygems/ext/builder.rb:99:in
  `run'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/2.6.0/rubygems/ext/builder.rb:51:in
  `block in make'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/2.6.0/rubygems/ext/builder.rb:43:in
  `each'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/2.6.0/rubygems/ext/builder.rb:43:in
  `make'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/2.6.0/rubygems/ext/ext_conf_builder.rb:62:in
  `block in build'
    /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/2.6.0/tempfile.rb:295:in `open'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/2.6.0/rubygems/ext/ext_conf_builder.rb:29:in
  `build'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/2.6.0/rubygems/ext/builder.rb:185:in
  `block in build_extension'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/2.6.0/monitor.rb:235:in
  `mon_synchronize'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/2.6.0/rubygems/ext/builder.rb:181:in
  `build_extension'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/2.6.0/rubygems/ext/builder.rb:229:in
  `block in build_extensions'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/2.6.0/rubygems/ext/builder.rb:226:in
  `each'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/2.6.0/rubygems/ext/builder.rb:226:in
  `build_extensions'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/2.6.0/rubygems/installer.rb:830:in
  `build_extensions'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/gems/2.6.0/gems/bundler-2.3.26/lib/bundler/rubygems_gem_installer.rb:72:in
  `build_extensions'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/gems/2.6.0/gems/bundler-2.3.26/lib/bundler/rubygems_gem_installer.rb:28:in
  `install'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/gems/2.6.0/gems/bundler-2.3.26/lib/bundler/source/rubygems.rb:207:in
  `install'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/gems/2.6.0/gems/bundler-2.3.26/lib/bundler/installer/gem_installer.rb:54:in
  `install'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/gems/2.6.0/gems/bundler-2.3.26/lib/bundler/installer/gem_installer.rb:16:in
  `install_from_spec'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/gems/2.6.0/gems/bundler-2.3.26/lib/bundler/installer/parallel_installer.rb:186:in
  `do_install'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/gems/2.6.0/gems/bundler-2.3.26/lib/bundler/installer/parallel_installer.rb:177:in
  `block in worker_pool'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/gems/2.6.0/gems/bundler-2.3.26/lib/bundler/worker.rb:62:in
  `apply_func'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/gems/2.6.0/gems/bundler-2.3.26/lib/bundler/worker.rb:57:in
  `block in process_queue'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/gems/2.6.0/gems/bundler-2.3.26/lib/bundler/worker.rb:54:in
  `loop'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/gems/2.6.0/gems/bundler-2.3.26/lib/bundler/worker.rb:54:in
  `process_queue'
  /opt/hostedtoolcache/Ruby/2.6.10/x64/lib/ruby/gems/2.6.0/gems/bundler-2.3.26/lib/bundler/worker.rb:91:in
  `block (2 levels) in create_threads'
  
  An error occurred while installing mathematical (1.6.14), and Bundler cannot
  continue.
  
  In Gemfile:
    asciidoctor-mathematical was resolved to 0.3.5, which depends on
      mathematical
  Took  31.00 seconds
Error: Error: The process '/opt/hostedtoolcache/Ruby/2.6.10/x64/bin/bundle' failed with exit code 5
    at ExecState._setResult (/home/runner/work/_actions/ruby/setup-ruby/v1/dist/index.js:5340:25)
    at ExecState.CheckComplete (/home/runner/work/_actions/ruby/setup-ruby/v1/dist/index.js:5323:18)
    at ChildProcess.<anonymous> (/home/runner/work/_actions/ruby/setup-ruby/v1/dist/index.js:5217:27)
    at ChildProcess.emit (node:events:390:28)
    at maybeClose (node:internal/child_process:1064:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:301:5)
```

**Comment by jnk0le** [comment]
and now it turns out that the wavedrom-cli doesn't accept `0b1010101` constants like the online editor
